assignments/assignment01.rb

```diff
@@ -38,7 +40,25 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  # your implementation here
+
+  # => PSEUDO CODE
```

Comment:
You are making this more complicated than it needs to be. Instead of looking at this character-by-character, think about words. You want to capitalize each word. How do you get the words in the string? Just do string.split. Then you can capitalize each word, then join the words back together.